lib/protocol.d.ts

```diff
@@ -1798,9 +1798,10 @@ declare namespace ts.server.protocol {
     namespace JsxEmit {
         type None = "None";
         type Preserve = "Preserve";
+        type ReactNative = "ReactNative";
```

Comment:
can you revert the changes to this file, the file is generated as part of the build. you have edited the correct file under src/server/protocol.ts